Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -538,9 +540,18 @@ static void BeginField(FieldType field)
 			  "(VI->BeginField): Address: %.08X | WPL %u | STD %u | ACV %u | Field %s",
 			  xfbAddr, m_PictureConfiguration.WPL, m_PictureConfiguration.STD,
 			  m_VerticalTimingRegister.ACV, fieldTypeNames[field]);
-
+	
 	if (xfbAddr)
 		g_video_backend->Video_BeginField(xfbAddr, fbWidth, fbStride, fbHeight);
+	
+	//printf("%u\n", m_VerticalTimingRegister.ACV);
+	VIWidth = (2 * m_HTiming0.HLW) - (m_HTiming0.HLW - m_HTiming1.HBS640)
+		- m_HTiming1.HBE640;
+	VIHeight = 2 * m_VerticalTimingRegister.ACV;
+	if (m_DisplayControlRegister.FMT == 1)
+		VMode = true;	//PAL50
```

Comment:
Please don't comment on the right like this (at least, i think that is the dolphin policy)